src/handlers/payout/post.ts

```diff
@@ -119,7 +129,139 @@ export const incentivizeComments = async () => {
   logger.info(`Skipping to generate a permit url for missing accounts. fallback: ${JSON.stringify(fallbackReward)}`);
 
   await addCommentToIssue(comment, issue.number);
-};
+}
+
+export async function incentivizePullRequestReviews() {
```

Comment:
same here
```suggestion
export const incentivizePullRequestReviews = async () => {
```

Reply:
@wannacfuture I made a new issue, #665,  so you can revert these functions to be anonymous so we can merge this in.